### PR TITLE
Add test for the macro nall

### PR DESCRIPTION
**Pull request by vmpyr**
Added a test in test/cartesian.jl for a macro nall at Base.Cartesian

### CODE REVIEW

**Comment by vmpyr** [comment]
Can someone please review this PR?

File: test/cartesian.jl [hunk 1]
```diff
@@ -515,6 +515,11 @@ end
 f39705() = Base.Cartesian.@nany 0 _ -> true
 @test f39705() === false
 
+@testset "Cartesian @nall macro test" begin
+    nall_test = Base.Cartesian.@nall 0 _ -> true
```
Comment (KristofferC) [inline test/cartesian.jl:519]: Could perhaps make this a bit more "difficult" and similar to real world usage, something like:

```julia
i_1, i_2, i_3 = 1, 2, 3;
@test Base.Cartesian.@nall 2 d->(i_d <= 2)
@test !Base.Cartesian.@nall 3 d->(i_d <= 2)

**Comment by KristofferC** [comment]
> Can someone please review this PR?

There is no need to bump PRs after so short time.

**Comment by vmpyr** [comment]
> There is no need to bump PRs after so short time.

I'm sorry, got a little carried away.

**Comment by vmpyr** [comment]
What should I do then? Merge PRs into one or make PRs only after certain time intervals? I am just starting out and was excited about contributing as much as possible

**Comment by sostock** [comment]
> What should I do then? Merge PRs into one or make PRs only after certain time intervals? I am just starting out and was excited about contributing as much as possible

Kristoffer wasn’t talking about submitting several PRs in a short time (which is fine), he was talking about the comment you posted shortly after submitting the PR:

> Can someone please review this PR?

This is referred to as "bumping" and is generally fine if you think that your PR has been forgotten or not been reviewed *for a long time*. It is unrealistic to expect that every PR is reviewed within a few hours after submitting it, so reminding people of your PR so shortly after submitting is unnecessary.

**Comment by vmpyr** [comment]
> Kristoffer wasn’t talking about submitting several PRs in a short time (which is fine), he was talking about the comment you posted shortly after submitting the PR

Oh, I'm sorry I misunderstood. I thought he meant not making many PRs in a short interval of time.

> This is referred to as "bumping" and is generally fine if you think that your PR has been forgotten or not been reviewed for a long time.

I did not know that. Thanks a lot for the info! I'll keep it in mind and not bump unnecessarily.

**Review by timholy — APPROVED** [review]

**Pull request merged**